exception
===================================

Custom exceptions for pyconll. These errors are a ``ParseError`` and a ``FormatError``.

A ``ParseError`` occurs when the source input to a CoNLL component is invalid, and a ``FormatError`` occurs when the internal state of the component is invalid, and the component cannot be output to a CoNLL string.


API
----------------------------------
.. automodule:: pyconll.exception
    :members:
    :special-members:
    :exclude-members: __dict__, __weakref__
